File: csharp/unittest-practice/tests/TortillaHarinaTest.cs
using NUnit.Framework;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using unittestPractice.main;

namespace unittestPractice.test
{
    [TestFixture]
    class TortillaHarinaTest
    {
        private TortillaHarina _tortillaHarina;

        [SetUp]
        public void SetUp()
        {
            _tortillaHarina = new TortillaHarina();
        }

        [TestCase]
        public void TestCurrentTemperature()
        {
            _tortillaHarina.SetCurrentTemperature(21);
            Assert.AreEqual(21, _tortillaHarina.GetCurrentTemperature());
        }

        [TestCase]
        public void TestFalseToast()
        {
            _tortillaHarina.Toast(false);
            Assert.IsFalse(_tortillaHarina.IsToasted());
        }

        [TestCase]
        public void TestTrueToast()
        {
            _tortillaHarina.Toast(true);
            Assert.IsTrue(_tortillaHarina.IsToasted());
        }

        [TestCase]
        public void TestToasting()
        {
            Assert.AreEqual(20, _tortillaHarina.GetToastTemperature());
        }
    }
    


}

File: python/src/test/TortillaHarinaTest.py
from TortillaHarina import *
import unittest

class TortillaHarinaTest(unittest.TestCase):
    def setUp(self):
        self.tortillaHarina = TortillaHarina()

    def test_CurrentTemperature(self):
        self.tortillaHarina.setCurrentTemperature(21)
        self.assertEqual(21,self.tortillaHarina.getCurrentTemperature())

    def test_FalseToast(self):
        self.tortillaHarina.toast(False)
        self.assertFalse(self.tortillaHarina.isToasted())

    def test_TrueToast(self):
        self.tortillaHarina.toast(True)
        self.assertTrue(self.tortillaHarina.isToasted())

    def test_Toasting(self):
        self.assertEqual(10,self.tortillaHarina.getToastTemperature())